playbooks/roles/nomad/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
```

Comment:
In it's current form, this file isn't particularly useful. :)

Reply:
Yeah I used `ansible-galaxy init` to generate this. This is one of the things to polish.